packages/salesforcedx-vscode-apex/src/codecoverage/statusBarToggle.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Disposable, StatusBarItem, window } from 'vscode';
+
+export class StatusBarToggle implements Disposable {
+  private static readonly toggleCodeCovCommand =
+    'sfdx.force.apex.toggle.colorizer';
+  private static readonly showIcon = '$(tasklist)';
+  private static readonly hideIcon = '$(three-bars)';
+  private static readonly toolTip = 'Apex Code Coverage highlighter';
```

Comment:
Text showed when you hover the status bar icon.

Reply:
Other nearby hover text seems to use imperative verbs and title-case capitalization. How about this? 'Highlight Apex Code Coverage'